packages/@aws-cdk-testing/framework-integ/test/aws-logs/test/integ.metricfilter-unit.ts.snapshot/manifest.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
It looks like these are all `ts` snapshots, probably a result of running
integ-runner with `--language typescript`. Can you remove these? We probably
should add a gitignore rule for this as well.